shared/references/cce-redhat-avail.txt

```diff
@@ -1,21 +1,3 @@
-CCE-86008-0
```

Comment:
It's usually better if you pick IDs from the middle of the archive so the chance of clashing with someone else that is also proposing a PR with new CCEs is very low.

Reply:
I know, but thanks to reinforce. It was intentional because I am working on other rules which need CCEs. By the way, mathematically, the chances of clashing with other PRs is the same if everyone is picking IDs from the middle. So, there is no safe rule for this and it is not also a big deal if a clash eventually happen. :))

Reply:
the point isn't to take it from the middle, but take it from a place that other people don't take it from, which means if everybody take it from the beginning they will conflict, if everyone will take it from the (exact) middle they will also conflict, so ideally take it from a random line :D

Reply:
It seems we are all aware of these corner cases. I wouldn't be so picky about it. It's not a big deal. If a conflict occurs, we resolve it easily. IMO, it's not worth giving these controversial guidelines to contributors as it depends on behaviors and could confuse them more than help them. But I agree that it is good to highlight the `fix_rules.py` script. And if someone manually picks ids from the list, we can consider they know what they're doing. :D